targets/targets.json

```diff
@@ -2874,6 +2874,10 @@
             "MCU_STM32G0"
         ],
         "public": false,
+        "mbed_rom_start": "0x08000000",
+        "mbed_rom_size": "0x80000",
+        "mbed_ram_start": "0x20000000",
+        "mbed_ram_size": "0x24000",
```

Comment:
Maybe this part could be removed ?
(same info available with "device_name": "STM32G0B1RETx")

Reply:
Yes, thanks, it works without it now. I added it when I had a non-matching device name with an index.json file and there were some errors related to these attributes.